### DISCUSSION TITLE
Error: CocoaPods's specs repository is too out-of-date to satisfy dependencies.

### DISCUSSION THREAD

**Discussion by bajpangosh** [codemagic-ci-cd/codemagic-docs#1564] [Q&A] [answered]
I'm new to using codemagic.io and I'm having trouble building an iOS app from my Github repository. I linked my repository, but I'm encountering an error during the build process:

"Error: CocoaPods's specs repository is too out-of-date to satisfy dependencies. To update the CocoaPods specs, run: pod repo update Error running pod install Build failed :| Failed to build for iOS."

Can you please advise me on how to resolve this issue?

**Comment by himesh-cm** [comment]
Hi @bajpangosh, please delete the `podfile` before your run `pod install` and try again. It may be related to a conflicting pub package version.

**Comment by bajpangosh** [comment]
SAME 

Error output from CocoaPods:
↳

    [!] Automatically assigning platform `iOS` with version `12.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error: CocoaPods's specs repository is too out-of-date to satisfy dependencies.
To update the CocoaPods specs, run:
  pod repo update

Error running pod install


Build failed :|
Failed to build for iOS

**Reply by himesh-cm** [reply]
Hi @bajpangosh, please share a build id.

**Reply by bajpangosh** [reply]
here it is.

https://codemagic.io/app/643abe5b51a13c973cc5f93f/build/643ac0856110060aa9e7b8c7

**Reply by himesh-cm** [reply]
checking

**Reply by himesh-cm** [reply, marked as answer]
It has conflicting dependencies, please update them and you should be fine.
```
[!] CocoaPods could not find compatible versions for pod "Firebase/CoreOnly":
      In snapshot (Podfile.lock):
        Firebase/CoreOnly (= 10.3.0)

      In Podfile:
        firebase_core (from `.symlinks/plugins/firebase_core/ios`) was resolved to 2.9.0, which depends on
          Firebase/CoreOnly (= 10.7.0)
```          

You could try updating the version of Firebase/CoreOnly in your Podfile to a version that is compatible with both the `Podfile.lock` file and the firebase_core.

**Reply by bajpangosh** [reply]
wow. i just deleted Podfile.lock. everything working. thank you.
